src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Setup/SetupComponentProvider.cs

```diff
@@ -21,6 +21,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Setup;
 /// </remarks>
 [Export(ExportContractNames.Scopes.UnconfiguredProject, typeof(IProjectDynamicLoadComponent))]
 [AppliesTo(ProjectCapability.DotNet)]
+[ExportInitialBuildRulesSubscriptions(SuggestedWorkload.SchemaName)]
```

Comment:
Both language service and NuGet restore code is also loaded by default in VS Green. I wonder they should be moved to a common location (or we need similar work for VS Green as well.)

Reply:
I'd vote to copy the attribute to green, given the rest of the code was largely a copy. Then if the targets change between implementations for any reason, they can be listed separately. But either approach would be fine.

Reply:
sure, I think we can add the export metadata to the language service rule in a separate PR.